### PR TITLE
Feature/conf

### PR DESCRIPTION
**Pull request by ARR552**
### What does this PR does?

This PR adds logs to show which conf parameters are loaded from env and file. Also, shows the configuration that has been loaded in hermez node.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Include tests
- [X] Respect code style and lint
- [ ] Update swagger file (if needed)
- [ ] Update documentation (*.md) (if needed)

### CODE REVIEW

**Review by arnaubennassar — APPROVED** [review]

**Review by Mikelle — APPROVED** [review]

File: config/config.go [hunk 1]
```diff
@@ -434,13 +438,18 @@ func LoadNode(path string, coordinator bool) (*Node, error) {
...
 }
 
 // LoadAPIServer loads the APIServer configuration from path.
 func LoadAPIServer(path string, coordinator bool) (*APIServer, error) {
-	var cfg APIServer
-	err := configLibrary.LoadConfig(path, DefaultValues, &cfg)
+	var cfg, aux APIServer
+	err := configLibrary.SourceParamsHezNode(path, &cfg, &aux)
+	if err != nil {
+		log.Println("Error SourceParamsHezNode: ", err.Error())
```
Comment (Mikelle) [inline config/config.go:450]: Is it supposed to only print this error and not return?
Maybe for this case we should use log.Error()?

File: config/config.go [hunk 2]
```diff
@@ -434,13 +438,18 @@ func LoadNode(path string, coordinator bool) (*Node, error) {
 			return nil, tracerr.Wrap(fmt.Errorf("error validating configuration file: %w", err))
 		}
 	}
+	log.Printf("Loaded Configuration: %+v", cfg)
```
Comment (mfcastellani) [inline config/config.go:441]: Will be great if this print "loaded configuration bla bla bla from config file|env|defaults"

**Review by mfcastellani — CHANGES_REQUESTED** [review]

File: config/config.go (hunk 1 above)
Comment (ARR552) [reply config/config.go:450]: I can't use log.Error, I'm using the standard log library

File: config/config.go (hunk 2 above)
Comment (ARR552) [reply config/config.go:441]: This functionality is in the config library. In this line what i show is the full configuration that the hermez node has loaded. Some parameter can be loaded fron env, others from file and finally others can be the default values

Comment (jeffprestes) [reply config/config.go:441]: But this is the point. We MUST know where the parameter data came from, each one of them. Not only the error.

Comment (ARR552) [reply config/config.go:441]: Yes, This is exactly what it does. Let me add a screen shot that shows how it looks like
image
In this screen shot you can see that four parameters are loaded using env and others using the file. Also it shows the full configuration where you can see the full configuration and default values.

Comment (jeffprestes) [reply config/config.go:441]: Thanks for sharing the image.

**Review by mfcastellani — APPROVED** [review]

**Pull request merged**